bamboo/docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
     environment:
       ES_JAVA_OPTS: "-Xms750m -Xmx750m"
   localstack:
-    image: localstack/localstack:0.10.7
+    image: localstack/localstack:0.10.9
```

Comment:
Updated this (but not to 11, I've added a Jira ticket for it) while trying to run down a connection issue locally.     As tests pass and there are a few bugs fixed, leaving it in without objection.